DDSim/DDSim/DD4hepSimulation.py

```diff
@@ -636,6 +654,52 @@ def __addParametersToRunHeader( self ):
 
     return runHeader
 
+
+  def __addParametersToEvent(self, parameters):
+    """ extract event parameters from the command line, to write in the header
+    of each event.
+    
+    :param parameters: list of command line parameters to parse
+    """
+    print parameters
+    stringParameters, intParameters, floatParameters, allParameters = {}, {}, {}, []
+    for p in parameters:
+      parameterAndValue = p.split( "=", 1 )
+      if len(parameterAndValue) is not 2:
```

Comment:
`is` should be used to compare identify of objects, not for numbers or strings, better use `!=`